proposed/longhorn/latest/templates/deployment-ui.yaml

```diff
@@ -33,7 +36,11 @@ metadata:
   name: longhorn-frontend
```

Comment:
need to rename the service name to http in order to make rancher-proxy works.

Reply:
Thank you, that did the trick!